<launch>

  <include file="$(find phantomx_arm_turtlebot_moveit_config)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>

  <node name="kinematicSolver" pkg="phantomx_arm_turtlebot_moveit_config" type="kinematicSolver" respawn="false" output="screen">
    <rosparam command="load" file="$(find phantomx_arm_turtlebot_moveit_config)/config/kinematics.yaml"/>
  </node>

</launch>
